src/plugins/gs-plugin-eos.c

```diff
@@ -401,6 +460,38 @@ gs_plugin_adopt_app (GsPlugin *plugin, GsApp *app)
 	gs_app_set_management_plugin (app, gs_plugin_get_name (plugin));
 }
 
+static void
+gs_plugin_eos_refine_app_branch (GsPlugin *plugin, GsApp *app)
+{
+	const char *default_branch;
+	const char *branch = NULL;
+	const char *origin = gs_app_get_origin (app);
+	GsPluginData *priv = NULL;
+
+	if (!gs_plugin_eos_app_is_flatpak (app))
+		return;
+
+	priv = gs_plugin_get_data (plugin);
+	default_branch = (char *) g_hash_table_lookup (priv->default_branches,
```

Comment:
There should be no need for the cast here, also because `default_branch` is `const char *`